application/models/watch.go

```diff
@@ -147,15 +159,31 @@ func (w *Watch) Destroy(tx *pop.Connection) error {
 	return tx.Destroy(w)
 }
 
-func (w *Watch) Meeting(tx *pop.Connection, user User) (*Meeting, error) {
+func (w *Watch) LoadMeeting(tx *pop.Connection, user User) error {
 	if w == nil || !w.MeetingID.Valid || user.ID != w.OwnerID {
-		return nil, nil
+		w.Meeting = nil
+		return nil
 	}
 	meeting := &Meeting{}
 	if err := tx.Find(meeting, w.MeetingID); err != nil {
-		return nil, err
+		return err
+	}
+	w.Meeting = meeting
+	return nil
+}
+
+// LoadForAPI assumes w.Destination and w.Origin were alread hydrated by an Eager query
```

Comment:
Why couldn't the Meeting be included in the Eager query?

Reply:
:+1: